packages/frontend/apps/electron/test/main/recording-effect.spec.ts

```
@@ -124,9 +124,31 @@ function createWorkspaceRef() {
   const blobSet = vi.fn(async () => 'blob-1');
   const openDoc = vi.fn();
   const createdDocs = new Set<string>();
+  type MockBlockRecord = { model: unknown } | null;
+  type MockBlockStore = {
+    addBlock: (
+      flavour: string,
+      props: Record<string, unknown>,
+      parentId?: string
+    ) => string;
+    getBlock: (id: string) => MockBlockRecord;
+    getBlock$: (id: string) => MockBlockRecord;
```

ESLint

[error] Finnish notation should not be used here.

(rxjs/finnish)